src/plugins/intel_cpu/tests/functional/behavior/ov_plugin/properties.cpp

```diff
@@ -190,20 +194,27 @@ TEST_F(OVClassConfigTestCPU, smoke_PluginSetConfigEnableProfiling) {
     ASSERT_EQ(enableProfiling, value);
 }
 
-const auto bf16_if_supported       = InferenceEngine::with_cpu_x86_bfloat16() ?    ov::element::bf16 : ov::element::f32;
+#if defined(OV_CPU_ARM_ENABLE_FP16)
+    const auto expected_precision_for_performance_mode = ov::element::f16;
+    const auto expected_precision_for_accuracy_mode = ov::element::f16;
```

Comment:
Why it is equal `f16`? I would expect the plugin to use f32 in case of accurayc hint.

Reply:
fixed